metadatastore/conf.py

```diff
@@ -67,10 +67,10 @@ def load_configuration(name, prefix, fields):
             config[field] = int(config[field])
 
     missing = [k for k, v in config.items() if v is None]
-    if missing:
-        raise KeyError("The configuration field(s) {0} "
-                       "were not found in any file or environmental "
-                       "variable.".format(missing))
+#    if missing:
```

Comment:
All fields in config will be set to None. As we enforce type in these fields, any connection attempt will fail prior to any attempt to connect to a database